source/backend/tests/unit/api/Services/ProjectServiceTest.cs

```diff
@@ -143,5 +143,60 @@ public async void Search_GetPage_Success()
             result.Should().NotBeNull();
             repository.Verify(x => x.GetPageAsync(It.IsAny<ProjectFilter>()), Times.Once);
         }
+
+        [Fact]
+        public void Add_Project_ShouldFail_NotAuthorized()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission();
+            var service = helper.Create<ProjectService>(user);
+
+            var repository = helper.GetService<Mock<IProjectRepository>>();
+
+            // Act
+            Action result = () => service.Add(new PimsProject());
+
+            // Assert
+            result.Should().Throw<NotAuthorizedException>();
+            repository.Verify(x => x.Add(It.IsAny<PimsProject>()), Times.Never);
+        }
+
+        [Fact]
+        public void Add_Project_ShouldFail_Filter_IsNull()
```

Comment:
There is no filter in this case. Add_Project_ShouldFail_Project_IsNull

Reply:
Name updated.